src/System.Globalization.Calendars/tests/Misc/Calendars.netstandard1.7.cs

```diff
@@ -0,0 +1,93 @@
+            yield return new object[] { new JapaneseCalendar()          , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new JapaneseLunisolarCalendar() , 0     , CalendarAlgorithmType.LunisolarCalendar   };
+            yield return new object[] { new JulianCalendar()            , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new KoreanCalendar()            , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new KoreanLunisolarCalendar()   , 0     , CalendarAlgorithmType.LunisolarCalendar   };
+            yield return new object[] { new PersianCalendar()           , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new TaiwanCalendar()            , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new TaiwanLunisolarCalendar()   , 0     , CalendarAlgorithmType.LunisolarCalendar   };
+            yield return new object[] { new ThaiBuddhistCalendar()      , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new UmAlQuraCalendar()          , 0     , CalendarAlgorithmType.LunarCalendar       };
+        }
+
+        [Theory]
+        [MemberData(nameof(Calendars_TestData))]
+        [PlatformSpecific(Xunit.PlatformID.Windows)] // Non Windows is not supported yet
+        public static void CloningTest(Calendar calendar, int yearHasLeapMonth, CalendarAlgorithmType algorithmType)
+        {
+            Calendar cloned = (Calendar) calendar.Clone();
+            Assert.Equal(cloned.GetType(), calendar.GetType());
+        }
+
+        [Theory]
+        [MemberData(nameof(Calendars_TestData))]
+        [PlatformSpecific(Xunit.PlatformID.Windows)] // Non Windows is not supported yet
+        public static void GetLeapMonthTest(Calendar calendar, int yearHasLeapMonth, CalendarAlgorithmType algorithmType)
+        {
+            if (yearHasLeapMonth > 0)
+            {
+                Assert.NotEqual(calendar.GetLeapMonth(yearHasLeapMonth),  0);
+                Assert.Equal(calendar.GetLeapMonth(yearHasLeapMonth - 1), 0);
+            }
+            else
+                Assert.True(calendar.GetLeapMonth(calendar.GetYear(DateTime.Today)) == 0);
+        }
+
+        [Theory]
+        [MemberData(nameof(Calendars_TestData))]
+        [PlatformSpecific(Xunit.PlatformID.Windows)] // Non Windows is not supported yet
+        public static void ReadOnlyTest(Calendar calendar, int yearHasLeapMonth, CalendarAlgorithmType algorithmType)
+        {
+            Assert.False(calendar.IsReadOnly);
+            var readOnlyCal = Calendar.ReadOnly(calendar);
+            Assert.True(readOnlyCal.IsReadOnly);
+            var colnedCal = (Calendar) readOnlyCal.Clone();
+            Assert.False(colnedCal.IsReadOnly);
```

Comment:
Especially on unix where we don't currently get line numbers in stack traces, it's helpful in Assert.True/False calls to include a message that helps to differentiate them when there are multiple such asserts in the same method.  Otherwise, when one fails, we don't know which.